engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -972,6 +943,52 @@ else if (allocatedContainers.isEmpty() && numRequestedContainers == 0 && dnmgr.c
     finishApplication(finalStatus, numTotalContainers);
   }
 
+  public ResourceRequestHandler createResourceRequestor()
```

Comment:
Made these methods public because I want to mock these methods with mockito, which only allows mocking public methods.